File: Programmers/src/제일작은수제거하기.java
class Solution {
    public int[] solution(int[] arr) {
        
        if(arr.length<=1){
            int[] answer = {-1};
            return answer;
        }
        
        int[] answer = new int[arr.length-1];
        int min=Integer.MAX_VALUE;
        int cnt=0;
        
        for(int i=0; i<arr.length; i++){
            min = Math.min(min, arr[i]);
        }
        
        for(int i=0; i<arr.length; i++){
            if(arr[i]==min)
                continue;
            
            answer[cnt++]=arr[i];
        }
        
        return answer;
    }
}

File: Programmers/src/Javascript/Lv1/제일작은수제거하기.js
function solution(arr) {
    
    let num = 0;
    for(let i=0; i<arr.length; i++){
        if(arr[num] > arr[i])
            num = i;
    }
    
    arr.splice(num, 1);
    if(arr.length===0)
        return [-1];
    
    return arr;
}